particle/particle/convert.py

```diff
@@ -0,0 +1,145 @@
+# This is a conversion file
```

Comment:
A full docstring in triple quotes seems more appropriate, with details on how to use it . simple example for the 2018 CSV file, say.

Reply:
+1